src/GitHubCreateMergePRs/Program.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        // Default when no args passed in.
+        var isDryRun = true;
+        var isAutomated = false;
+        var githubToken = string.Empty;
+
+        foreach (var arg in args)
+        {
+            if (arg.StartsWith("--isDryRun"))
```

Comment:
Nit: should arguments be case sensitive as are done here?

Reply:
I'm not quite understanding - It is case sensitive right now, would it be preferred for them to be case insensitive?